### ISSUE TITLE
Travis CI integration

### ISSUE THREAD

**Issue by gaul** [dkim0419/SoundRecorder#71]
It would be great if you configured Travis CI to build SoundRecorder.

**Comment by khokharnikunj8** [comment]
Added  in PR #73

**Comment by dkim0419** [comment]
cool, so is this something similar to team city? the project doesn't actually have any tests to run at the moment, but if we can add some in this seems great

**Comment by khokharnikunj8** [comment]
@dkim0419  yes . it is like team city to check all the builds of branch and pull requests.

**Comment by naXa777** [comment]
Travis CI is integrated in a fork -> https://github.com/naXa777/SoundRecorder
